pkg/io/video/convert.go

```diff
@@ -28,27 +28,44 @@ func imageToYCbCr(dst *image.YCbCr, src image.Image) {
 		i0 := 1 * flat
 		i1 := 2 * flat
 		i2 := 3 * flat
-		b := make([]uint8, i2)
-		dst.Y = b[:i0:i0]
-		dst.Cb = b[i0:i1:i1]
-		dst.Cr = b[i1:i2:i2]
+		if cap(dst.Y) < 3*flat {
+			dst.Y = make([]uint8, i2)
```

Comment:
Realloc only if it lacks capacity.

Reply:
knit: could you make it `i2` instead of `3*flat`?